instrumentation/logger.go

```diff
@@ -13,6 +13,24 @@ import (
 	"os"
 )
 
+func GetBootstrapCrashLogger() log.Logger {
+	path := "./orbs-network-bootstrap.log"
+
+	logFile, err := os.OpenFile(path, os.O_APPEND|os.O_CREATE|os.O_WRONLY, 0644)
+	if err != nil {
+		panic(err)
+	}
+
+	fileWriter := log.NewTruncatingFileWriter(logFile)
+	outputs := []log.Output{
+		log.NewFormattingOutput(fileWriter, log.NewHumanReadableFormatter()),
```

Comment:
why write to a file at all? docker stdout seems to be the direction we're going to, no?

Reply:
Regardless of our plans, we are currently writing to a file, so bootstrap errors should currently go to that file as well.